lib/hanami/slice_registrar.rb

```diff
@@ -75,6 +75,12 @@ def to_a
       slices.values
     end
 
+    def with_nested
+      to_a.map do |slice|
+        [slice] + slice.slices.with_nested
+      end.flatten
```

Comment:
Mind making this change, @waiting-for-dev? Just keeps the brace usage consistent with the rest of the file, and `flat_map` lets us remove the trailing `flatten`.

```suggestion
      to_a.flat_map { |slice|
        [slice] + slice.slices.with_nested
      }
```

Reply:
Good point. Done!